test/smoke/tsconfig.json

```diff
@@ -11,6 +11,7 @@
 		"outDir": "out",
 		"sourceMap": true,
 		"skipLibCheck": true,
+		"declaration": true,
```

Comment:
Probably should surround this with 
// --- Start Positron ---
// --- End Positron ---

Reply:
Good catch, thank you!